lamindb/integrations/_vitessce.py

```diff
@@ -39,9 +39,12 @@ def save_vitessce_config(vitessce_config: VitessceConfig, description: str) -> A
             if "url" not in file:
                 raise ValueError("Each file must have a 'url' key.")
             filename = file["url"].split("/")[-1]
-            assert filename.endswith(
+            if not filename.endswith(
```

Comment:
Great that we'll get a test for this!

Reply:
The whole function is only tested via notebooks actually at the moment.